content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md

```diff
@@ -1,1335 +1,968 @@
-to Windows.
+Key Kubernetes elements work the same way in Windows as they do in Linux. This
+section refers to several key workload enablers and how they map to Windows.
 
 * [Pods](/docs/concepts/workloads/pods/)
 
-  A Pod is the basic building block of Kubernetes–the smallest and simplest
-  unit in the Kubernetes object model that you create or deploy. You may not
-  deploy Windows and Linux containers in the same Pod. All containers in a Pod
-  are scheduled onto a single Node where each Node represents a specific
-  platform and architecture. The following Pod capabilities, properties and
-  events are supported with Windows containers:
+  A Pod is the basic building block of Kubernetes–the smallest and simplest unit in
+  the Kubernetes object model that you create or deploy. You may not deploy Windows and
+  Linux containers in the same Pod. All containers in a Pod are scheduled onto a single
+  Node where each Node represents a specific platform and architecture. The following
+  Pod capabilities, properties and events are supported with Windows containers:
 
   * Single or multiple containers per Pod with process isolation and volume sharing
-  * Pod status fields
+  * Pod `status` fields
   * Readiness and Liveness probes
   * postStart & preStop container lifecycle events
   * ConfigMap, Secrets: as environment variables or volumes
-  * EmptyDir
+  * `emptyDir` volumes
   * Named pipe host mounts
   * Resource limits
-* [Controllers](/docs/concepts/workloads/controllers/)
-
-  Kubernetes controllers handle the desired state of Pods. The following
-  workload controllers are supported with Windows containers:
-
+* [Workload resources](/docs/concepts/workloads/controllers/) including:
   * ReplicaSet
-  * ReplicationController
   * Deployments
   * StatefulSets
   * DaemonSet
   * Job
   * CronJob
+  * ReplicationController
+* {{< glossary_tooltip text="Services" term_id="service" >}}\
+  See [Load balancing and Services](#load-balancing-and-services) for more details.
 
-* [Services](/docs/concepts/services-networking/service/)
-
-  A Kubernetes Service is an abstraction which defines a logical set of Pods
-  and a policy by which to access them - sometimes called a micro-service. You
-  can use services for cross-operating system connectivity. In Windows, services
-  can utilize the following types, properties and capabilities:
-
-  * Service Environment variables
-  * NodePort
-  * ClusterIP
-  * LoadBalancer
-  * ExternalName
-  * Headless services
-
-Pods, Controllers and Services are critical elements to managing Windows
+Pods, workload resources, and Services are critical elements to managing Windows
 workloads on Kubernetes. However, on their own they are not enough to enable
 the proper lifecycle management of Windows workloads in a dynamic cloud native
-environment. We added support for the following features:
+environment. Kubernetes also supports:
```

Comment:
I'm not sure what the original text meant to imply!

Reply:
I think maybe these functionalities were not available when Windows container support went stable (v1.14) but has since been added?

I wonder if we should just remove parts of this section because I am also not sure what the original text meant to imply.